Relecture du WSDL SAV3.0

mamraoui opened the pull request:
Bonjour,

à la relecture de cette proposition, voilà les retours de BouyguesTélécom :

1. il manque dans le WSDL la spécification des champs
	- _escalation_
	- _description_
2. _GRTRecoveryRange_ est plutôt une liste de 2 valeurs possibles et non pas une simple string
3. Il y a 2 specification du type "appointmentMode"; il faut garder uniquement la spéc avec définition de liste
4. pour certains champs il est autorisé que leurs longueur soit 0
	- _serviceProviderID_ 		=> c'est un num SIREN qui a une longueur fixe = 9
	- _serviceProviderName_	=> le nom de l'OC ne devrait pas être vide si le champ est présent dans le flux
	- _supplierID_ 			=> c'est un num SIREN qui a une longueur fixe = 9
	- _supplierName_		=> le nom de l'OI ne devrait pas être vide si le champ est présent dans le flux
	- commercialId		=> si ce champ est présent dans le flux il ne doit pas être vide
	- technicalId			=> si ce champ est présent dans le flux il ne doit pas être vide

Merci.
Moëz

dlevasseuraltitude committed:
feat: commit initial avec nouveaux champs

dlevasseuraltitude committed:
fix: suppression fichier inutile

dlevasseuraltitude committed:
fix: champ en lower camel case

dlevasseuraltitude committed:
fix: lowerCamelCase

dlevasseuraltitude commented:
Bonjour Moëz,

Je suis sur le point de pousser ces corrections, mais j'ai tout de même une question sur le champ description.

D'après la documentation, ce champ fait parti des champs commun de l'OSSJ. Il est déjà présent dans plusieurs fichiers, notamment les fichiers OSSJ-Common-CBEBi-v1-5.xsd et OSSJ-Common-CBECore-v1-5.xsd.

Dois-je donc quand même l'ajouter au fichier FTTHMUTTroubleTicketValue-v3-0.xsd?

Merci.
David

dlevasseuraltitude committed:
fix: correction suite retour mamraoui

- Ajout de la description
- Ajout de l'escalation
- Mise à jour des contraintes sur les champs serviceProviderId, supplierId, supplierName et serviceProviderName
- Modification de la contrainte du GRTRecoveryRange et de l'appointmentMode

mamraoui commented:
Bonjour @dlevasseuraltitude ,

je suis d'accord avec vous, il ne faut pas en rajouter une autre déclaration du champ "description" (c'était une erreur de ma part).

Cdlt.
Moëz

dlevasseuraltitude committed:
fix: Suppression description

Suppression description suite retour mamraoui

dlevasseuraltitude commented:
Bonjour @mamraoui ,

Je viens de commit de nouvelles modifications, pourriez-vous me dire si cela vous semble mieux?

Merci,

Cordialement,
David

mamraoui commented:
@dlevasseuraltitude : une petit coquille au niveau de la déclaration du champ "GRTRecoveryRange" (2ème valeur = INCONDITIONNELLE plutôt que SIGNALISATION).
A part ça, tout le reste est ok pour moi :)

Merci.
Moëz

dlevasseuraltitude committed:
fix: corection GRTRecoveryRange

- modification valeurs possibles: doublon SIGNALISATION remplacé par
SIGNALISATION et INCONDITIONNELLE

dlevasseuraltitude commented:
@mamraoui , effectivement je n'avais pas vu celle ci. C'est corrigé :)

Merci,
David
